.github/dependabot.yml

```diff
@@ -0,0 +1,64 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: 'npm'
```

Comment:
Above each of these separate parts of the app, let's put comment lines, and a newline after the settings:

```yml
# Root package.json
...

# Next Front-End
...

# Autodeployment Server
...
```